flow/connectors/postgres/cdc.go

```diff
@@ -182,9 +184,49 @@ func (p *PostgresCDCSource) consumeStream(
 			}
 
 			if rec != nil {
-				result.Records = append(result.Records, rec)
+				// check if the record has unchanged toast columns.
+				hasUnchangedToastColumns := rec.HasUnchangedToastColumns()
+
+				tableName := rec.GetTableName()
+				pkeyCol := req.TableNameSchemaMapping[tableName].PrimaryKeyColumn
```

Comment:
add a comment saying that this is the destination table name and that it should ideally be the source table name.